src/www/app/fake_tunnel.ts

```diff
@@ -28,40 +28,36 @@ export class FakeOutlineTunnel implements cordova.plugins.outline.Tunnel {
   }
 
   private playUnreachable() {
-    return !(this.config.name && this.config.name.toLowerCase().includes('unreachable'));
+    return this.config.name && this.config.name.toLowerCase().includes('unreachable');
```

Comment:
while we're here, `this.config.name?.toLowerCase()...`

Reply:
Done in 3e74749. Updated typescript to v3.7, which introduced optional support. I'll keep an eye out to use optionals where possible.